packages/gatsby-theme-api-docs/src/components/type/properties/rows/name-type.js

```diff
@@ -39,8 +32,15 @@ const NameType = ({
         ) : null}
       </p>
       <p className="name-type">
-        <span className="name">{dataForTypeToLinkTo.displayPrefix}</span>
-        {typeof type === 'string' ? <span className="name">{type}</span> : type}
+        {typeToRender.displayPrefix && (
+          <span className="name">{typeToRender.displayPrefix}</span>
+        )}
+
+        {typeof typeToRender.type === 'string' ? (
+          <span className="name">{typeToRender.type}</span>
+        ) : (
+          typeToRender.type
```

Comment:
I'm confused. If `typeToRender.type` is not a string, what can it be?

Reply:
A `<Link/>`

Reply:
I see...kind of confusing by whatever